src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -162,6 +163,30 @@ export class PiaIntakeService {
 
       whereClause.push(...additionalWhereClauses);
     }
+
+    /** filter logic here */
+    if (query.filterByStatus) {
+      whereClause.forEach((clause) => {
+        clause.status = PiaIntakeStatusEnum[query.filterByStatus];
```

Comment:
shouldn't this be `clause.status = query.filterByStatus`?

Reply:
it throw an error

<img width="1082" alt="Screen Shot 2023-01-24 at 4 08 38 PM" src="https://user-images.githubusercontent.com/57013495/214449260-157bc59d-4a80-4078-9d7d-f2bf67bcbb3e.png">